### DISCUSSION TITLE
working on a particular version of a room

### DISCUSSION THREAD

**Discussion by LovesPictures** [mozilla/hubs#2709] [General Questions & Discussion] [answered]
Hello, 
Are you able to tell me how I continue to work on a room when it's been published. What I mean in particulate is that I have issued a link to a room so need to work on that particular version of the room.   Any ideas?

**Comment by SeanBannister** [comment, marked as answer]
When you're in a room you can click the menu button in the top left, there's an option "Room and Scene Info", a popup will display and clicking on the name of the room will take you to the scene page where you can click "Edit in Spoke". Note that if the scene is owned by someone else and they didn't set it as "remixable" you won't be able to edit in Spoke.

Take a look at [this video in the docs](https://hubs.mozilla.com/docs/spoke-creating-projects.html#remixing-scenes-from-hubs) which show you how to do it.

**Comment by LovesPictures** [comment]
Hi Sean,

Thank you for your answer, I figured it out in the end, and have updated the room.  Thank you for the information, I'll take a look.
